aptos-move/framework/aptos-framework/releases/artifacts/current/build/AptosFramework/sources/TokenTransfers.move

```diff
@@ -0,0 +1,138 @@
+/// This module provides the foundation for transferring of Tokens
+module AptosFramework::TokenTransfers {
+    use Std::GUID::ID;
+    use Std::Signer;
+    use AptosFramework::Table::{Self, Table};
+    use AptosFramework::Token::{Self, Token};
+
+    struct TokenTransfers<TokenType: copy + drop + store> has key {
+        pending_transfers: Table<address, Table<ID, Token<TokenType>>>,
+    }
+
+    public fun initialize_token_transfers<TokenType: copy + drop + store>(account: &signer) {
+        move_to(
+            account,
+            TokenTransfers<TokenType> {
+                pending_transfers: Table::create<address, Table<ID, Token<TokenType>>>(),
+            }
+        )
+    }
+
+    // Make an entry into pending transfers and extract from gallery
+    public fun transfer_to<TokenType: copy + drop + store>(
```

Comment:
we probably should call it differently, it's more like escrow/prepare

Reply:
I was liking the name "offer_to" and "claim_from"

Reply:
offer_to sounds good to me